src/fields/commonFieldTypes.ts

```diff
@@ -11,6 +11,8 @@ export interface CommonFieldProps extends MarginProps {
   required?: boolean
   error?: boolean
   errorMsg?: string
+  showStatus?: boolean
+  status?: 'complete' | null
```

Comment:
This approach will let us have different statuses without having to adjust the previous ones. We would potentially need to create an object to store the styles for each status if we decide to add more. 

Just to clarify. 
`showStatus` displays an empty space so that there is no layout shift it appears
'status' defines the status and when it's null there is just space

Reply:
We could possibly simplify this and make it complete specific. As currently you can pass it any status and it would render `Complete`. What do you think?